lib/cli/commands/create.ts

```diff
@@ -25,13 +25,13 @@ const formatOutput = (response: CreateNodeResponse.AsObject) => {
       console.log(`The following lnd wallets were initialized: ${response.initializedLndsList.join(', ')}`);
     }
     if (response.initializedRaiden) {
-      console.log('The wallet for raiden was initialized');
+      console.log('The wallet for raiden was initialized.');
     }
 
     console.log(`
-Please write down your 24 word mnemonic. It will allow you to recover your xud
-key and funds for the initialized wallets listed above should you forget your
-password or lose your device. Keep it somewhere safe, it is your ONLY backup.
+Please write down your 24 word mnemonic. It will allow you to recover your xud node key and
+on-chain funds for the initialized wallets listed above, should you forget your password or
```

Comment:
Minor grammar nit but I don't think the comma after "above" is necessary.

Reply:
Removed.